openslides_backend/action/actions/organization/initial_import.py

```diff
@@ -0,0 +1,104 @@
+    Should be callable from the management service.
+    """
+
+    model = Organization()
+    schema = DefaultSchema(Organization()).get_default_schema(
+        additional_required_fields={"data": {"type": "object"}},
+        title="Import initial data.",
+        description="Import an initial data json in an empty datastore.",
+    )
+
+    def perform(
+        self, action_data: ActionData, user_id: int, internal: bool = False
+    ) -> Tuple[Optional[WriteRequest], Optional[ActionResults]]:
+        """
+        Simplified entrypoint to perform the action.
+        """
+        self.user_id = user_id
+        self.index = 0
+        action_data = self.get_updated_instances(action_data)
+        instance = next(iter(action_data))
+        self.validate_instance(instance)
+        try:
+            self.check_permissions(instance)
+        except MissingPermission as e:
+            msg = f"You are not allowed to perform action {self.name}."
+            e.message = msg + " " + e.message
+            raise e
+        instance = self.base_update_instance(instance)
+        self.write_requests.extend(self.create_write_requests(instance))
+        final_write_request = self.process_write_requests()
+        result = [self.create_action_result_element(instance)]
+        return (final_write_request, result)
+
+    def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        data = instance["data"]
+
+        self.check_empty_datastore()
+
+        # check datavalidation
+        checker = Checker(data=data, mode="all")
+        try:
+            checker.run_check()
+        except CheckException as ce:
+            raise ActionException(str(ce))
+
+        return instance
+
+    def check_empty_datastore(self) -> None:
+        filter_ = FilterOperator("id", ">=", 1)
+        check_collections = ["user", "organization", "committee", "meeting"]
+        for collection in check_collections:
+            if self.datastore.exists(
+                Collection(collection),
+                filter_,
+                DeletedModelsBehaviour.ALL_MODELS,
+                False,
+            ):
+                raise ActionException("Datastore is not empty.")
+
+    def create_write_requests(self, instance: Dict[str, Any]) -> Iterable[WriteRequest]:
+        json_data = instance["data"]
+        write_requests = []
+        for collection in json_data:
+            for entry in json_data[collection].values():
+                fqid = FullQualifiedId(Collection(collection), entry["id"])
+                write_requests.append(
+                    self.build_write_request(
+                        EventType.Create,
+                        fqid,
+                        "initial import",
+                        entry,
+                    )
+                )
+        return write_requests
+
+    def create_action_result_element(
+        self, instance: Dict[str, Any]
+    ) -> Optional[ActionResultElement]:
+        return {"id": 1}
```

Comment:
I don't think a result is necessary here since it's alway 1. You can just delete this method and return [None] above as results